braindecode/models/eegresnet.py

```diff
@@ -0,0 +1,243 @@
+# Authors: Robin Tibor Schirrmeister
+#          Tonio Ball
+#
+# License: BSD-3
+
+import numpy as np
+from torch import nn
+from torch.nn import init
+import torch as th
+from torch.nn.functional import elu
+
+from braindecode.models.base import BaseModel
+
+from braindecode.models.modules import Expression, AvgPool2dWithConv
+from braindecode.models.functions import identity
+from braindecode.models.modules import np_to_var
+
+
+class EEGResNet(BaseModel):
+    """
+    Residual Network for EEG.
```

Comment:
for new code let's force ourself to add docstrings

Reply:
it should also have been added to the api.rst page

Reply:
hope docstrings are ok/better now?